example/Makefile

```diff
@@ -103,4 +126,93 @@ stop-lvmd:
 		rm -f $(BACKING_STORE)/backing_store; \
 	fi
 
-.PHONY: launch-kind shutdown-kind start-lvmd stop-lvmd run setup clean
+$(TMPDIR)/scheduler/kube-scheduler.yaml: migrate/kube-scheduler.yaml
+	mkdir -p $(TMPDIR)/scheduler
+	cp $< $@
+
+.PHONY: migration-test
+migration-test: build/topolvm.io_logicalvolumes.yaml
+	$(MAKE) run BUILD_IMAGE=true USE_LEGACY=true
+
+	# Stopping legacy topolvm
+	$(KUBECTL) -n topolvm-system delete deploy topolvm-controller topolvm-cert-manager topolvm-cert-manager-cainjector topolvm-cert-manager-webhook
+	$(KUBECTL) -n topolvm-system delete ds topolvm-node topolvm-scheduler
+	$(KUBECTL) delete storageclass topolvm-provisioner topolvm-provisioner-thin
+	$(KUBECTL) delete csidriver topolvm.cybozu.com
+	$(KUBECTL) delete clusterrole \
```

Comment:
Don't you need to remove ServiceAccount resources?
Are they reused after migration has completed?

Reply:
> Are they reused after migration has completed?

Sorry, but namespace is finally removed. So it looks good.